spark/src/main/scala/org/apache/spark/sql/comet/execution/shuffle/CometShuffleExchangeExec.scala

```diff
@@ -454,12 +458,13 @@ class CometShuffleWriteProcessor(
     new SQLShuffleWriteMetricsReporter(context.taskMetrics().shuffleWriteMetrics, metrics)
   }
 
+  // FIXME: we need actually prev of rdd?
```

Comment:
What does it mean?

Reply:
We used to have this code `rdd.asInstanceOf[MapPartitionsRDD[_, _]].prev.asInstanceOf[RDD[ColumnarBatch]]`

But new Spark 4.0 interface do not pass rdd anymore. So we cannot access `prev`